includes/class-genesis-responsive-slider-admin.php

```diff
@@ -0,0 +1,432 @@
+					</p>
+
+				<hr class="div" />
+
+				<h4><?php esc_html_e( 'Content Settings', 'genesis-responsive-slider' ); ?></h4>
+
+					<p>
+						<input type="checkbox" name="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_no_link]" id="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_title_show]" value="1" <?php checked( 1, Genesis_Responsive_Slider::genesis_get_responsive_slider_option( 'slideshow_no_link' ) ); ?> /> <label for="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_no_link]"><?php esc_html_e( 'Do not link Slider image to Post/Page.', 'genesis-responsive-slider' ); ?></label>
+					</p>
+
+					<p>
+						<input type="checkbox" name="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_title_show]" id="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_title_show]" value="1" <?php checked( 1, Genesis_Responsive_Slider::genesis_get_responsive_slider_option( 'slideshow_title_show' ) ); ?> /> <label for="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_title_show]"><?php esc_html_e( 'Display Post/Page Title in Slider?', 'genesis-responsive-slider' ); ?></label>
+					</p>
+					<p>
+						<input type="checkbox" name="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_excerpt_show]" id="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_excerpt_show]" value="1" <?php checked( 1, Genesis_Responsive_Slider::genesis_get_responsive_slider_option( 'slideshow_excerpt_show' ) ); ?> /> <label for="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_excerpt_show]"><?php esc_html_e( 'Display Content in Slider?', 'genesis-responsive-slider' ); ?></label>
+					</p>
+
+					<p>
+						<input type="checkbox" name="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_hide_mobile]" id="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_hide_mobile]" value="1" <?php checked( 1, Genesis_Responsive_Slider::genesis_get_responsive_slider_option( 'slideshow_hide_mobile' ) ); ?> /> <label for="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_hide_mobile]"><?php esc_html_e( 'Hide Title & Content on Mobile Devices', 'genesis-responsive-slider' ); ?></label>
+					</p>
+
+					<p>
+						<?php esc_html_e( 'Select one of the following:', 'genesis-responsive-slider' ); ?>
+						<select name="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_excerpt_content]" id="<?php echo esc_html( GENESIS_SETTINGS_FIELD ); ?>[slideshow_excerpt_content]">
```

Comment:
As above, I believe GENESIS_SETTINGS_FIELD should be GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD